### ISSUE TITLE
avoid using std::regex in watershell.h

### ISSUE THREAD

**Issue by b4sh1t1** [RITRedteam/watershell-cpp#3]
Hello,

I notice issue while compiling in old gcc 4.8.9, which is effect compiling in old version, can we avoid using regex in watershell.h line 31 ?

[root@domain watershell-cpp]# g++ main.cpp watershell.cpp -o watershell
In file included from /usr/include/c++/4.8.2/regex:35:0,
                 from watershell.h:31,
                 from main.cpp:1:
/usr/include/c++/4.8.2/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^
In file included from /usr/include/c++/4.8.2/regex:35:0,
                 from watershell.h:31,
                 from watershell.cpp:1:
/usr/include/c++/4.8.2/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^

[root@domain watershell-cpp]# g++ --version
g++ (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

[root@domain watershell-cpp]#

**Comment by hulto** [comment]
Have you tried building statically with the provided docker container?

**Comment by b4sh1t1** [comment]
No, i did fix this issue by installing new gcc and defind the LD_PATH, but i face another issue with ( watershell.service )... after connection and systemd working fine, the backdoor connected but dont show result from command, ex:

```
└──╼ $python watershell-cli.py -t ip -p 12345 -i
Connecting to Watershell on ('185.63.253.216', 12345)...
Connected!
//\\watershell//\\>> id

//\\watershell//\\>> 
```

the wateshell.service

[Unit]
Description=persistence

```
[Service]
Type=simple

ExecStart=/bin/watershell -l 12345

[Install]
WantedBy=multi-user.target
```

Now if i run the backdoor from root user in the terminal, the backdoor controller is work fine.. but when backdoor started with watershell.service, the backdoor connect but with no result back ?!?
